script.js

```diff
@@ -7,54 +7,77 @@ let displayedValue = 0;
 let isOperationClicked = false;
 let operation = '';
 
-numbers.forEach((number) => {
-  number.addEventListener('click', (e) => {
-    if (isOperationClicked) {
-      display.textContent = '';
-      isOperationClicked = false;
-    }
-    if (display.textContent === '0') {
-      display.textContent = '';
+const dataInput = (value) => {
```

Comment:
It is a function, so please rename it to define what exactly this function does.